src/stp/new_constants/ControlConstants.cpp

```diff
@@ -17,7 +17,7 @@ constexpr double MAX_CHIP_POWER = 8;
 constexpr double MIN_CHIP_POWER = 1.01;
 
 // Dribbler constants
-const double TURN_ON_DRIBBLER_DISTANCE = 3 * ROBOT_RADIUS;
+const double TURN_ON_DRIBBLER_DISTANCE = 5 * ROBOT_RADIUS;
```

Comment:
The naming does not yet explain me the meaning of the variable. So additional documentation is needed here.